src/sentry/api/serializers/base.py

```diff
@@ -1,27 +1,27 @@
-from typing import Any, Mapping, Optional, Sequence, Union
+from typing import Any, Callable, List, Mapping, MutableMapping, Optional, Union
 
 import sentry_sdk
 from django.contrib.auth.models import AnonymousUser
 
 registry = {}
 
 
-def register(type: Any):
+def register(type: Any) -> Callable[[Any], Any]:
     """ A wrapper that adds the wrapped Serializer to the Serializer registry (see above) for the key `type`. """
 
-    def wrapped(cls):
+    def wrapped(cls: Any) -> Any:
         registry[type] = cls()
         return cls
 
     return wrapped
 
 
 def serialize(
-    objects: Union[Any, Sequence[Any]],
+    objects: Union[Any, List[Any]],
```

Comment:
Any reason for changing away from Sequence?

Reply:
I was debugging a different issue and this got left in :(